src/js/forms.js

```diff
@@ -161,7 +161,7 @@ object.getFormTrackingManager = function (core, trackerId, contextAdder) {
 
 					lodash.forEach(innerElementTags, function (tagname) {
 						lodash.forEach(form.getElementsByTagName(tagname), function (innerElement) {
-							if (fieldFilter(innerElement) && !innerElement[trackingMarker]) {
+							if (fieldFilter(innerElement) && !innerElement[trackingMarker] && innerElement.type !== 'password') {
```

Comment:
Great point - I suspect there will also be instances (perhaps generated or copied HTML) where `type="PASSWORD"`